### ISSUE TITLE
Error compiling on raspbian

### ISSUE THREAD

**Issue by flashbackpa** [raboof/nethogs#111]
Hello, i have the following error compiling source code on raspbian (debian derivative Linux raspi5 4.4.37-v7+ #936 SMP Fri Dec 9 16:56:49 GMT 2016 armv7l GNU/Linux):

make -C src -f MakeApp.mk decpcap_test
make[1]: Entering directory '/opt/nethogs/src'
cc  -Wall -Wextra -c decpcap.c
g++  -Wall -Wextra -Wno-missing-field-initializers  decpcap_test.cpp decpcap.o -o decpcap_test -lpcap -lm
make[1]: Leaving directory '/opt/nethogs/src'
make -C src -f MakeApp.mk test
make[1]: Entering directory '/opt/nethogs/src'
g++ -Wall -Wextra -Wno-missing-field-initializers    conninode_test.cpp   -o conninode_test
for test in conninode_test; do echo $test ; ./$test ; done
conninode_test
make[1]: Leaving directory '/opt/nethogs/src'
make -C src -f MakeApp.mk nethogs
make[1]: Entering directory '/opt/nethogs/src'
g++  -Wall -Wextra -Wno-missing-field-initializers -c packet.cpp
g++  -Wall -Wextra -Wno-missing-field-initializers -c connection.cpp
g++  -Wall -Wextra -Wno-missing-field-initializers -c process.cpp
g++  -Wall -Wextra -Wno-missing-field-initializers -c cui.cpp -DVERSION=\"\"
g++  -Wall -Wextra -Wno-missing-field-initializers -c inode2prog.cpp
g++  -Wall -Wextra -Wno-missing-field-initializers -c conninode.cpp
g++ -Wall -Wextra -Wno-missing-field-initializers   -c -o devices.o devices.cpp
g++  -Wall -Wextra -Wno-missing-field-initializers  main.cpp packet.o connection.o process.o decpcap.o cui.o inode2prog.o conninode.o devices.o -o nethogs -lpcap -lm -lncurses -DVERSION=\"\"
main.cpp: In function 'bool wait_for_next_trigger()':
main.cpp:104:39: warning: narrowing conversion of 'refreshdelay' from 'unsigned int' to '__time_t {aka long int}' inside { } is ill-formed in C++11 [-Wnarrowing]
     timeval timeout = {refreshdelay, 0};
                                       ^
main.cpp: In function 'int main(int, char**)':
main.cpp:286:42: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if (last_refresh_time + refreshdelay <= now) {
                                          ^
make[1]: Leaving directory '/opt/nethogs/src'

**Comment by raboof** [comment]
Interesting - these seem to be warnings, are you sure they make the build fail for you?

(of course it'd be nice to fix the warnings anyway :) )

**Comment by flashbackpa** [comment]
No, compilation end successfully, they were just warnings as you tought...

**Comment by raboof** [comment]
Do you still get them on latest git master? I suspect https://github.com/raboof/nethogs/pull/113 might have fixed them!
